lib/Duo/API.pm

```diff
@@ -181,10 +200,10 @@ sub api_call {
 
     if (grep(/^$method$/, qw(POST PUT))) {
         $req->header('Content-type' => 'application/x-www-form-urlencoded');
-        $req->content($canon_params);
+        $req->content($request_params);
     }
     else {
-        $path .= '?' . $canon_params;
+        $path .= '?' . $request_params;
     }
 
     $req->uri('https://' . $self->{'host'} . $path);
```

Comment:
I'd love to see a test that proves that we are signing things with the sorted params, but still sending the unsorted params for the request

Reply:
I can make that happen!